servers/visual/rasterizer.cpp

```diff
@@ -360,7 +360,7 @@ void RasterizerStorage::multimesh_instance_set_transform(RID p_multimesh, int p_
 				static int32_t warn_count = 0;
 				warn_count++;
 				if (((warn_count % 2048) == 0) && GLOBAL_GET("debug/settings/physics_interpolation/enable_warnings")) {
-					WARN_PRINT("Interpolated MultiMesh transform should usually be set during physics process (possibly benign).");
+					WARN_PRINT("Called on interpolated multimesh from outside physics process (possibly benign).");
```

Comment:
Not sure if "Called on ..." will be clear to users, maybe it should give more context?

How about:
```suggestion
					WARN_PRINT("[Physics interpolation] MultiMesh interpolation is being triggered from outside physics process, this might lead to issues (possibly benign).");
```